backend/cfg/cfg_dataflow.mli

```diff
@@ -73,10 +73,10 @@ end
 module type Backward_transfer = sig
   type domain
 
-  val basic : domain -> exn:domain -> Cfg.basic Cfg.instruction -> domain
+  val basic : domain -> exn:domain -> Cfg.basic Cfg.instruction -> domain * Cfg.basic Cfg.instruction
 
   val terminator :
-    domain -> exn:domain -> Cfg.terminator Cfg.instruction -> domain
+    domain -> exn:domain -> Cfg.terminator Cfg.instruction -> domain * Cfg.terminator Cfg.instruction
 
```

Comment:
ouch, I haven't anticipated that making "live" field immutable would cause this change, sorry. This interface is awkward and doesn't match "textbook" analysis. As discussed offline, the issue is that the granularity of the transformers is per instruction, whereas the result of the analysis is only per bock.